### PR TITLE
mydumper: init at 0.9.3

### PR DESCRIPTION
**Pull request by Izorkin**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/backup/mydumper/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  name = "mydumper-${version}";
+
+  src = fetchFromGitHub {
+    owner  = "maxbube";
+    repo   = "mydumper";
+    rev    = "eef4d8291803e9ba57b4e58458cec3f4f7f0b203";
+    sha256 = "139v6707sxyslg7n1fii8b1ybdq50hbqhc8zf6p1cr3h2hhl6ns9";
+  };
+
+  buildInputs = [ cmake glib mariadb.client.dev libressl pcre pkgconfig zlib ];
```
Comment (adisbladis) [inline pkgs/tools/backup/mydumper/default.nix]: `cmake` and `pkgconfig` should be under `nativeBuildInputs` instead

File: pkgs/tools/backup/mydumper/default.nix [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  meta = with stdenv.lib; {
```
Comment (adisbladis) [inline pkgs/tools/backup/mydumper/default.nix]: Are you willing to maintain this package?
If so please add yourself as a maintainer in https://github.com/NixOS/nixpkgs/blob/master/lib/maintainers.nix

Personally I'm very reluctant to merge new packages without a maintainer.

File: pkgs/tools/backup/mydumper/default.nix [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+}:
+
+stdenv.mkDerivation rec {
+  version = "0.9.3";
+  name = "mydumper-${version}";
+
+  src = fetchFromGitHub {
+    owner  = "maxbube";
+    repo   = "mydumper";
+    rev    = "eef4d8291803e9ba57b4e58458cec3f4f7f0b203";
```
Comment (adisbladis) [inline pkgs/tools/backup/mydumper/default.nix]: Please change to `rev = "v${version}";`.

**Review by adisbladis — CHANGES_REQUESTED** [review]

File: pkgs/tools/backup/mydumper/default.nix (hunk 1 above)
Comment (Izorkin) [reply pkgs/tools/backup/mydumper/default.nix]: if changed to - nativebuildInputs = [ cmake pkgconfig ];
error build - make: *** No rule to make target 'install'.  Stop.

**Comment by adisbladis** [comment]
I have rebased and squashed your commits (usually a new package would be one commit, any fixup commits should be squashed).

Thank you!

**Pull request merged**